sqlglot/parser.py

```diff
@@ -1352,7 +1352,7 @@ def _parse_drop(self, exists: bool = False) -> exp.Drop | exp.Command:
             exp.Drop,
             comments=start.comments,
             exists=exists or self._parse_exists(),
-            this=self._parse_table(schema=True),
+            this=self._parse_table(schema=True, is_db_reference=kind == "SCHEMA"),
```

Comment:
what happens if schema is not capitalized?

Reply:
why not just check the token type like you do in create?

Reply:
Updated: https://github.com/tobymao/sqlglot/blob/76999173554117381a15884698fcbd4126e65125/sqlglot/parser.py#L1355-L1357